astropy/utils/iers/tests/test_iers.py

```diff
@@ -330,22 +335,16 @@ def test_IERS_B_parameters_loading_into_IERS_Auto():
                      "correctly to IERS Auto".format(name)))
 
 
-# Issue with FTP, rework test into previous one when it's fixed
-@pytest.mark.xfail('TRAVIS')
 @pytest.mark.remote_data
-def test_iers_a_dl():
-    iersa_tab = iers.IERS_A.open(iers.IERS_A_URL, cache=False)
+@pytest.mark.parametrize('url', (iers.IERS_A_URL, iers.IERS_A_URL_MIRROR))
+def test_iers_a_dl(url):
     try:
-        # some basic checks to ensure the format makes sense
-        assert len(iersa_tab) > 0
-        assert 'UT1_UTC_A' in iersa_tab.colnames
-    finally:
-        iers.IERS_A.close()
-
-
-@pytest.mark.remote_data
-def test_iers_a_dl_mirror():
-    iersa_tab = iers.IERS_A.open(iers.IERS_A_URL_MIRROR, cache=False)
+        iersa_tab = iers.IERS_A.open(url, cache=False)
+    except URLError:
+        if os.environ.get('TRAVIS') and url == iers.IERS_A_URL:
+            pytest.xfail('FTP timed out on Travis CI')
```

Comment:
Just out of curiosity, can we increase the timeout in IERS_A.open?

Reply:
@eteq says it works fine from his machine, so there is also a chance that FTP services sometimes blocks Travis CI after certain threshold. I really don't know. 🤷‍♀  I can just revert this whole thing if you want. It is kinda out of scope anyway, as it resulted in xpass/xfail instead of warnings.

Reply:
Maybe let's indeed revert and deal with this separately. In my experience Travis also sometimes times out because the outside world connections can be slower.